webpack.config.js

```diff
@@ -0,0 +1,48 @@
+var Encore = require('@symfony/webpack-encore');
+
+Encore
+    // directory where compiled assets will be stored
+    .setOutputPath('views/build/')
+    // public path used by the web server to access the output path
+    .setPublicPath('/assets')
```

Comment:
this is wrong. the public path is `build`, not `assets` (as you don't use the manifest, you are less impacted by the wrong config of course)

Reply:
The compiled files should be relative to the the "output" option from the cli command.

Reply:
@codecov-io the current WebBuilder embeds every asset in the html code (css, js, images), so it's not possible for those to be relative to the "output-dir" option of the build command.

One thing I had in mind was making the build command to copy the compiled assets (or maybe symlink them if it's possible) to the "output-dir" and referencing them rather than embedding them. If you all want, I can give it a go and add this to the PR as well.